### PR TITLE
Add pictures to the best papers.

### PR DESCRIPTION
**Pull request by bangerth**
I *think* this should work, but don't have mkdocs installed on my work computer. Let's see what happens when merged -- the page hasn't been announced yet, so I can always just fix it tonight from my laptop.

### CODE REVIEW

File: docs/community/best_paper_award.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ The best deal.II-based paper from 2023 is the following:
 > [[DOI: 10.1016/j.cma.2023.115983]](http://doi.org/10.1016/j.cma.2023.115983)
 > [[URL]](https://www.sciencedirect.com/science/article/pii/S0045782523001068)
 > [[Preprint]](https://arxiv.org/abs/2207.12460)
+>
+> ![An image from the publication](../large-assets/best-paper-award/2023/figure_res_baseline_active_tension.png)
```
Comment (bangerth) [inline docs/community/best_paper_award.md:31]: OK, this link does not work. What is the correct link to reference these pictures? Does it have to be a global URL, or should I reference a relative path? In either case, which one is it?

Comment (tamiko) [reply docs/community/best_paper_award.md]: @bangerth Because it is `large_assets`. I have fixed the typo!

Comment (bangerth) [reply docs/community/best_paper_award.md]: Oh, good grief :-( I played with it for fifteen minutes last night and just couldn't figure it out. I totally missed that issue...

**Comment by bangerth** [comment]
But it still doesn't work: https://dealii.org/community/best_paper_award/

**Comment by bangerth** [comment]
OK, now it does. Perhaps the machine just needed to run the update script. All good now, in any case!

**Pull request merged**